traitcuration/traits/templates/traits/datasources.html

```diff
@@ -2,8 +2,14 @@
 {% block title %} External Datasources {%endblock %} 
 {% block content %}
 <div class="uk-container uk-container-xlarge">
-  <a href="{% url 'fetch_data' %}"
-    ><button class="uk-button uk-button-primary">FETCH DATA</button>
+  <a href="{% url 'clinvar_data' %}"
+    ><button class="uk-button uk-button-primary">CLINVAR DATA</button>
+  </a>
+  <a href="{% url 'zooma_suggestions' %}"
+    ><button class="uk-button uk-button-primary">ZOOMA SUGGESTIONS</button>
+  </a>
+  <a href="{% url 'dummy_data' %}"
+    ><button class="uk-button uk-button-primary">IMPORT DUMMY DATA</button>
```

Comment:
I suggest moving this button so that it comes first in the page, because otherwise my reflex is to click them in order, in which case I'll import ClinVar and ZOOMA, and then proceed to drop all that with dummy data import :)

Also, if it's not too hard to implement, I'd like to ask for a "IMPORT ALL button", which would trigger, in order: 
* Dummy data import
* ClinVar import
* ZOOMA import